docs/faq.rst

```diff
@@ -291,6 +292,70 @@ See :ref:`encrypted_repos` for more details.
           user
           <https://security.stackexchange.com/questions/14000/environment-variable-accessibility-in-linux/14009#14009>`_.
 
+Using ``BORG_PASSCOMMAND`` with a properly permissioned file
+  Another option is to create a file with a password in it in your home
+  directory and use permissions to keep anyone else from reading it. For
+  example, first create a key::
+
+    dd if=/dev/urandom of=~/.borg-passphrase bs=1024 count=1
```

Comment:
Perhaps use hex here as well, to avoid non-printables in the passphrase. Also, any NUL terminates the passphrase, so there is a relatively good chance that this passphrase is only a few characters long.